[RDY] vim-patch:7.4.2189

ckelsel opened the pull request:
vim-patch:7.4.2189
    
    https://github.com/vim/vim/commit/f71d7b9ee5ceba75f70c30845332ddd728fd16c6
    
    Author: Bram Moolenaar <Bram@vim.org>
    Date:   Tue Aug 9 22:14:05 2016 +0200
    
        patch 7.4.2189
        Problem:    Cannot detect encoding in a fifo.
        Solution:   Extend the stdin way of detecting encoding to fifo.  Add a t
                    for detecting encoding on stdin and fifo. (Ken Takata)

ckelsel committed:
vim-patch:7.4.2189

https://github.com/vim/vim/commit/f71d7b9ee5ceba75f70c30845332ddd728fd16c6

Author: Bram Moolenaar <Bram@vim.org>
Date:   Tue Aug 9 22:14:05 2016 +0200

    patch 7.4.2189
    Problem:    Cannot detect encoding in a fifo.
    Solution:   Extend the stdin way of detecting encoding to fifo.  Add a test
                for detecting encoding on stdin and fifo. (Ken Takata)

jamessan commented:
It would be good to include https://github.com/vim/vim/releases/tag/v7.4.2229 too.

ckelsel committed:
vim-patch:7.4.2229

https://github.com/vim/vim/commit/f04507d132fbcb63999167ec006fc6e700b5af4f

Author: Bram Moolenaar <Bram@vim.org>
Date:   Sat Aug 20 15:05:39 2016 +0200

    patch 7.4.2229
    Problem:    Startup test fails on Solaris.
    Solution:   Recognize a character device. (Danek Duvall)

ckelsel commented:
vim-patch:7.4.2229
    
    https://github.com/vim/vim/commit/f04507d132fbcb63999167ec006fc6e700b5af4f
    
    Author: Bram Moolenaar <Bram@vim.org>
    Date:   Sat Aug 20 15:05:39 2016 +0200
    
        patch 7.4.2229
        Problem:    Startup test fails on Solaris.
        Solution:   Recognize a character device. (Danek Duvall)

ckelsel committed:
lint
